test-data/unit/check-namedtuple.test

```diff
@@ -185,3 +185,35 @@ class C(B): pass
 B(1).b
 C(2).b
 [builtins fixtures/property.py]
+
+[case testNamedTupleAsDict]
+from collections import namedtuple
+
+X = namedtuple('X', ['x', 'y'])
+x = None  # type: X
+a = x._asdict()
```

Comment:
Great! Feel free to ask questions in this PR. To direct the work I recommend making up some test cases and then tracing through the code with pdb or PyCharm's debugger -- that's often how I do things. The README.md has some hints on how to run a specific test case using myunit, without the overhead of runtests.py.